src/sinks/util/sink.rs

```diff
@@ -66,10 +66,12 @@ use tracing_futures::Instrument;
 
 // === StreamSink ===
 
-#[async_trait]
-pub trait StreamSink {
-    async fn run(&mut self, input: BoxStream<'_, Event>) -> Result<(), ()>;
-}
+pub use vector_core::sink::StreamSink;
+
+// #[async_trait]
+// pub trait StreamSink {
+//     async fn run(&mut self, input: BoxStream<'_, Event>) -> Result<(), ()>;
+// }
```

Comment:
Does this need to stay here?

Reply:
That was a mistake. Corrected in 517057b.